### DISCUSSION TITLE
Sunsynk LifeLynk AIO inverter supported?

### DISCUSSION THREAD

**Discussion by deltamelter** [kellerza/sunsynk#205] [Q&A] [answered]
Sunsynk Lifelink is an all in one inverter and battery it has the "wifi logger" connection but don't see anything resembling RS485/modbus connector.
Is there any support for controlling these without being tied to the cloud?
![image](https://github.com/kellerza/sunsynk/assets/27061628/0fbffef3-ec7c-4a87-9472-f36049c5f15c)

**Comment by jeybee74** [comment]
Did you get this to work?
I guess you can use the port marked as BATTERY/BMS/CAN, this probably also has the pins with RS485 in it.

**Reply by Waz-Cpt** [reply, marked as answer]
Yes, Sunsynk and Deye has been making use of the 8 contacts in an RJ45 connector to provide both CAN and RS485 on a single connector. This should be outlined in the owners manual of the device.

**Reply by nethergatewws** [reply]
I'm trying to Frankenstein this now using a HF2221 and I'm not having any luck. The lifelynk S is what I'm trying.  I've managed to sort the 12kw 3 phase version of Sunsynk but this one is being a pain for me.  How did you wire it what pin outs did you use and which port ?  The CAN doesn't seem to work for me and neither does port 1 or 3.
